spec/services/imports/sales_logs_import_service_spec.rb

```diff
@@ -384,6 +384,42 @@ def open_file(directory, filename)
           sales_log = SalesLog.find_by(old_id: sales_log_id)
           expect(sales_log.proplen_asked).to eq(1)
         end
+
+        context "when setting soctenant fields" do
```

Comment:
I think we should test the different behaviour in the different years

Reply:
But let me know if there are cases that I missed

Reply:
ahh ok gotcha, nice!